fetch-content.sh

```diff
@@ -15,7 +15,28 @@ if [ ! -e "$DIR" ]; then
 	mkdir -p "$DIR"
 	pushd "$DIR"
 
-	wget http://xwis.net/downloads/Red-Alert-2-Multiplayer.exe
+	DOWNLOADERS=("wget" "curl")
+	DOWNLOAD_COMMANDS=("wget URL" "curl -o \"Red-Alert-2-Multiplayer.exe\" URL")
```

Comment:
Using `curl` this way will never show a failure.  `curl` will not surface a nonzero exit code if a file does not exist.  Example:

```
curl -o /dev/null http://example.com/does-not-exist.tar
echo $?
curl -f -o /dev/null http://example.com/does-not-exist.tar
echo $?
```